### PR TITLE
Support for Symfony 2.8

### PR DESCRIPTION
**Pull request by notFloran**
See #100

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,13 +12,13 @@
     ],
     "require": {
         "php": ">=5.3.0",
-        "symfony/framework-bundle": "2.*,<2.8",
-        "symfony/form": ">=2.1,<2.8",
+        "symfony/framework-bundle": "~2.1",
+        "symfony/form": "~2.1",
```
Comment (trsteel88) [inline composer.json]: Both form and framework-bundle should be >=2.1,<2.9

Comment (notFloran) [inline composer.json]: done

File: composer.json [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
-        "symfony/framework-bundle": "2.*,<2.8",
-        "symfony/form": ">=2.1,<2.8",
+        "symfony/framework-bundle": ">=2.1,<2.9",
+        "symfony/form": ">=2.1,<2.9",
         "twig/twig": ">=1.1,<2.0",
         "ezyang/htmlpurifier": "~4.0"
     },
     "require-dev": {
-        "symfony/twig-bundle": "2.*,<2.8",
+        "symfony/twig-bundle": "~2.1",
```
Comment (trsteel88) [inline composer.json]: 2.*,<2.9 here

Comment (notFloran) [inline composer.json]: Done, sorry for the mistakes

**Comment by trsteel88** [comment]
Thanks

**Pull request merged**